ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/Rejection.scala

```diff
@@ -86,7 +86,7 @@ object Rejection {
 
   final case class PartiesNotKnownOnLedger(parties: Iterable[Ref.Party]) extends Rejection {
     override lazy val description: String =
-      s"Parties not known on ledger: ${parties.mkString("[", ", ", "]")}"
+      s"Party not known on ledger: ${parties.mkString("[", ", ", "]")}"
```

Comment:
Plural seems more correct since we are talking about multiple

Reply:
7de9f2f9d008313c9b2960d420d4c8f182a2f65a